easy_my_coop_website/i18n/fr_BE.po

```diff
@@ -355,7 +355,7 @@ msgstr "Vous ne pouvez souscrire à des types de parts différents"
 
 #. module: easy_my_coop_website
 #: model_terms:ir.ui.view,arch_db:easy_my_coop_website.becomecooperator
-msgid "You have already an account?"
+msgid "Do you already have an account?"
 msgstr ""
 "Vous avez déjà un compte ? Cliquer ici pour vous connecter avant de remplir "
 "le formulaire de prises de parts."
```

Comment:
Wrong translation for source string.

Reply:
It is the good translation, although it is more precise than the original.  I am not going to change that, since it is not related to this task.

Reply:
OK! :thumbsup: